src/libraries/System.Formats.Tar/tests/TarWriter/TarWriter.WriteEntry.File.Tests.Windows.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using Microsoft.DotNet.XUnitExtensions;
+using Xunit;
+using Xunit.Sdk;
+
+namespace System.Formats.Tar.Tests;
+
+public partial class TarWriter_WriteEntry_File_Tests : TarWriter_File_Base
+{
+    [Theory]
+    [InlineData(TarEntryFormat.V7)]
+    [InlineData(TarEntryFormat.Ustar)]
+    [InlineData(TarEntryFormat.Pax)]
+    [InlineData(TarEntryFormat.Gnu)]
+    public void Add_Junction_As_SymbolicLink(TarEntryFormat format)
+    {
+        using TempDirectory root = new TempDirectory();
+        string targetName = "TargetDirectory";
+        string junctionName = "JunctionDirectory";
+        string targetPath = Path.Join(root.Path, targetName);
+        string junctionPath = Path.Join(root.Path, junctionName);
+
+        Directory.CreateDirectory(targetPath);
+
+        Assert.True(MountHelper.CreateJunction(junctionPath, targetPath));
+        DirectoryInfo junctionInfo = new(junctionPath);
+
+        using MemoryStream archive = new MemoryStream();
+        using (TarWriter writer = new TarWriter(archive, format, leaveOpen: true))
+        {
+            writer.WriteEntry(fileName: junctionPath, entryName: junctionPath);
+        }
+
+        archive.Position = 0;
+        using (TarReader reader = new TarReader(archive))
+        {
+            TarEntry entry = reader.GetNextEntry();
+            Assert.Equal(format, entry.Format);
+
+            Assert.NotNull(entry);
+            Assert.Equal(junctionPath, entry.Name);
+            Assert.Equal(targetPath, entry.LinkName);
+            Assert.Equal(TarEntryType.SymbolicLink, entry.EntryType);
+            Assert.Null(entry.DataStream);
+
+            VerifyPlatformSpecificMetadata(junctionPath, entry);
+
+            Assert.Null(reader.GetNextEntry());
+        }
+    }
+
+    [Theory]
```

Comment:
You need to use `[ConditionalTheory]` in order for SkipTestException to actually skip and not take down the whole test run.
```suggestion
    [ConditionalTheory]
```